deploy/kubernetes/base/init-pod.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  creationTimestamp: null
```

Comment:
what did this do? can you also add a mention about `creationTimestamp` in the PR description?

Reply:
I don't know what this does, but from what I read in the k8 documentation, empty and default fields in the configs do no need be be specified.

Reply:
Could you amend the PR description to mention this change as well?